src/demo/DocsDecorator/DocsDecorator.scss

```diff
@@ -93,4 +44,55 @@ $root: '.docs-decorator';
     .docs-example {
         margin: $offset-m 0;
     }
+
+    :not([class*='yc-']):not([class*='g-']) > {
```

Comment:
Added to prevent styles from affecting elements without classes inside components